### PR TITLE
update translation

### PR DESCRIPTION
**Pull request by JrDw0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,102 @@
...
+Vulhub is an open source vulnerability lab for public. You don't need any docker knowledge, just execute two simple commend that you can compile and run a complete vulnerability environment mirror.
 
 ## Installation
 
-在ubuntu16.04下安装docker/docker-compose:
+Install the docker/docker-compose under ubuntu16.04:
 
 ```bash
-# 安装pip
+# install pip
```
Comment (sfdye) [inline README.md]: Install

File: README.md [hunk 2]
```diff
@@ -1,104 +1,102 @@
...
 curl -s https://bootstrap.pypa.io/get-pip.py | python3
 
-# 安装最新版docker
+# Install the latest version docker
 curl -s https://get.docker.com/ | sh
 
-# 启动docker服务
+# run docker service
```
Comment (sfdye) [inline README.md]: Run

File: README.md [hunk 3]
```diff
@@ -1,104 +1,102 @@
...
 service docker start
 
-# 安装compose
+# install docker compose
```
Comment (sfdye) [inline README.md]: Install

File: README.md [hunk 4]
```diff
@@ -1,104 +1,102 @@
 # Vulhub - Docker-Compose files for vulnerabilities environment
 
+[中文版本(Chinese version)](README.zh-cn.md)
+
 [![Docker Stars](https://img.shields.io/badge/docker%20stars-20k-blue.svg)](https://hub.docker.com/u/vulhub/) [![GitHub](https://img.shields.io/github/license/vulhub/vulhub.svg)](https://github.com/vulhub/vulhub/blob/master/LICENSE)
  [![Chat on Discord](https://img.shields.io/discord/485505185167179778.svg)](https://discord.gg/GhMB3Z) [![Backers and sponors on Patreon](https://img.shields.io/badge/sponsor-patreon-73d6a1.svg)](https://www.patreon.com/phith0n) [![Backers and sponors on Opencollective](https://img.shields.io/badge/backer-opencollective-f89a76.svg)](https://www.patreon.com/phith0n)
 
-Vulhub是一个面向大众的开源漏洞靶场，无需docker知识，简单执行两条命令即可编译、运行一个完整的漏洞靶场镜像。
+Vulhub is an open source vulnerability lab for public. You don't need any docker knowledge, just execute two simple commend that you can compile and run a complete vulnerability environment mirror.
```
Comment (sfdye) [inline README.md]: I think "开源漏洞靶场" is the most important translation of all. We should make this right.

Comment (sfdye) [inline README.md]: No docker knowledge is required, just execute two simple commands and you have a vulnerable environment.

File: README.md [hunk 5]
```diff
@@ -1,104 +1,102 @@
...
 
 ## Installation
 
-在ubuntu16.04下安装docker/docker-compose:
+Install the docker/docker-compose under ubuntu16.04:
```
Comment (sfdye) [inline README.md]: on Ubuntu 16.04

File: README.md [hunk 6]
```diff
@@ -1,104 +1,102 @@
...
+# Run docker service
 service docker start
 
-# 安装compose
+# Install docker compose
 pip install docker-compose 
 ```
 
-其他操作系统安装docker和docker-compose可能会有些许不同，请阅读Docker文档进行安装。
+The installation steps of docker and docker-compose under others operating system may be slightly different, please read the docker documentation for installation.
```
Comment (sfdye) [inline README.md]: The installation steps of docker and docker-compose for others operating system might be slightly different, please refer to the [docker documentation](https://docs.docker.com/install/) for details.

File: README.md [hunk 7]
```diff
@@ -1,104 +1,102 @@
...
-测试完成后，删除整个环境
+There is a **README** document in each environment directory, please read this file for vulnerability/environment testing.
 
+After the test, delete the entire environment with the following command.
 ```
 docker-compose down -v
 ```
 
-本项目每个漏洞环境均附带文档，建议你购买1G内存的vps搭建漏洞测试环境，文档中所说的`your-ip`均指你的vps的ip地址，如果你是用虚拟机搭建测试环境，是指你的虚拟机IP，而不是docker容器内部的IP，请不要混淆。
+It is recommended to use VPS with 1G or above memory to build a vulnerability environment.It is recommended to use vps with 1G or above memory to build a vulnerability environment.The `your-ip` mentioned in the document refers to the IP address of your vps. If you are using a virtual machine to build a test environment, it refers to your virtual machine IP, not the IP inside the docker container, please don't confuse it.
```
Comment (sfdye) [inline README.md]: It is recommended to use a VPS of at least 1G memory to build a vulnerability environment. 

后面一句重复了

File: README.md [hunk 8]
```diff
@@ -1,104 +1,102 @@
 # Vulhub - Docker-Compose files for vulnerabilities environment
 
+[中文版本(Chinese version)](README.zh-cn.md)
```
Comment (phith0n) [inline README.md]: 这一句放图标下面吧

File: README.md [hunk 9]
```diff
@@ -1,104 +1,102 @@
...
 [![Docker Stars](https://img.shields.io/badge/docker%20stars-20k-blue.svg)](https://hub.docker.com/u/vulhub/) [![GitHub](https://img.shields.io/github/license/vulhub/vulhub.svg)](https://github.com/vulhub/vulhub/blob/master/LICENSE)
  [![Chat on Discord](https://img.shields.io/discord/485505185167179778.svg)](https://discord.gg/GhMB3Z) [![Backers and sponors on Patreon](https://img.shields.io/badge/sponsor-patreon-73d6a1.svg)](https://www.patreon.com/phith0n) [![Backers and sponors on Opencollective](https://img.shields.io/badge/backer-opencollective-f89a76.svg)](https://www.patreon.com/phith0n)
 
-Vulhub是一个面向大众的开源漏洞靶场，无需docker知识，简单执行两条命令即可编译、运行一个完整的漏洞靶场镜像。
+Vulhub is an open-source hack lab. No docker knowledge is required, just execute two simple commands and you have a vulnerable environment.
 
 ## Installation
 
-在ubuntu16.04下安装docker/docker-compose:
+Install the docker/docker-compose on ubuntu16.04:
```
Comment (phith0n) [inline README.md]: Ubuntu 16.04

**Pull request merged**